### PR TITLE
add ayushtamra

### LINKED ISSUE #517: Ayush Tamra
myGithub

### PR DESCRIPTION
**Pull request by ayushtamra**
resolve #517

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 1]
```diff
@@ -910,5 +910,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/75872316/118384765-6dad0480-b626-11eb-85ce-a6af24c19181.png",
     "issueNumber": 499
-  }
+  },
+  {
+    "name": "Ayush Tamra",
+    "githubUsername": "ayushtamra",
+    "imageUrl": "https://user-images.githubusercontent.com/64156683/118395408-fb5d1400-b667-11eb-99c9-549cc5e49b00.png",
+    "issueNumber": 517
+   }
```
Comment (schmelto) [inline data.json:919]:
```suggestion
  }
```

found one little issue please align this brace with the others :)

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (ayushtamra) [reply data.json]: Thanks for suggesting
I have resolved the required changes.

**Review by starlightknown — APPROVED** [review]
Looks good

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:

**Review by deephunt3r — APPROVED** [review]
LGTM 🚀

**Review by schmelto — APPROVED** [review]

**Pull request merged**